surveys/templates/location_detail.html

```diff
@@ -0,0 +1,57 @@
+{% extends 'base.html' %}
+{% load static %}
+{% load leaflet_tags %}
+{% load crispy_forms_tags %}
+
+{% block extracss %}
+  {% leaflet_css %}
```

Comment:
I'm not totally sure why, but the map loads at a height on 300px on my machine, which looks a little bit squished:

![Screenshot from 2019-06-05 14-21-44](https://user-images.githubusercontent.com/14170650/58983822-43e1d680-879d-11e9-8a6a-84fc4968eddb.png)

Reply:
This was the intended height, but now that you mention it does look a bit small. I'll make it longer!